### ISSUE TITLE
Website Issues @ Integramod

### ISSUE THREAD

**Issue by MWE001** [IntegraMOD/IntegraMOD151#91]
Hey Ven,

I tried leaving a post over at the Integramod website 2 times since your last one in December but as we see it is a mess error wise. Do you have PM's turned on over at phpBB2 Refugees? 

Feel free to delete this issue thread once you get this message.

**Comment by vendethiel** [comment]
Hey Ray,

I need to check my settings over there, I don’t remember if I tweaked them or not.

**Comment by MWE001** [comment]
Right on man. It's all good. I had no idea that we had to turn PM's (or DM's which ever one prefers now a days) on over there. I always thought PM's were native. I think I am forgetting more about BB's than I ever knew. No hurry on that. I just figured since Integra was erroring out and I could no longer reply there, a PM there or Icy even may be a good avenue in case we ever needed to drop each other a quick note.
